contracts/src/bridge/SequencerInbox.sol

```diff
@@ -61,6 +61,28 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
         _;
     }
 
+    uint256 internal immutable deployTimeChainId = block.chainid;
+
+    function _chainIdChanged() internal view returns (bool) {
+        return deployTimeChainId != block.chainid;
+    }
+
+    uint256 internal constant FORK_DELAY_BLOCKS = 1;
+    uint256 internal constant FORK_FUTURE_BLOCKS = 1;
+    uint256 internal constant FORK_DELAY_SECONDS = 1;
+    uint256 internal constant FORK_FUTURE_SECONDS = 1;
+
+    function _checkAndHandleFork() internal {
+        if (_chainIdChanged() && maxTimeVariation.delayBlocks != FORK_DELAY_BLOCKS) {
+            ISequencerInbox.MaxTimeVariation memory maxTimeVariation_;
+            maxTimeVariation_.delayBlocks = FORK_DELAY_BLOCKS;
+            maxTimeVariation_.futureBlocks = FORK_FUTURE_BLOCKS;
+            maxTimeVariation_.delaySeconds = FORK_DELAY_SECONDS;
+            maxTimeVariation_.futureSeconds = FORK_FUTURE_SECONDS;
+            maxTimeVariation = maxTimeVariation_;
```

Comment:
```suggestion
            maxTimeVariation = ISequencerInbox.MaxTimeVariation({delayBlocks, ...});
```

Reply:
changed to write directly to storage instead